From c18ae75852b1340ca502528138bf421659f61a3d Mon Sep 17 00:00:00 2001
From: Joey Hess <joey@kitenet.net>
Date: Mon, 15 Apr 2013 12:44:15 -0400
Subject: [PATCH] remove TH

Should not need these icons in git-annex, so not worth using the Evil
Splicer.
---
 Network/Wai/Application/Static.hs | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/Network/Wai/Application/Static.hs b/Network/Wai/Application/Static.hs
index 3195fbb..b48aa01 100644
--- a/Network/Wai/Application/Static.hs
+++ b/Network/Wai/Application/Static.hs
@@ -33,8 +33,6 @@ import Control.Monad.IO.Class (liftIO)
 
 import Blaze.ByteString.Builder (toByteString)
 
-import Data.FileEmbed (embedFile)
-
 import Data.Text (Text)
 import qualified Data.Text as T
 
@@ -198,8 +196,6 @@ staticAppPieces _ _ req
         H.status405
         [("Content-Type", "text/plain")]
         "Only GET is supported"
-staticAppPieces _ [".hidden", "folder.png"] _  = return $ W.responseLBS H.status200 [("Content-Type", "image/png")] $ L.fromChunks [$(embedFile "images/folder.png")]
-staticAppPieces _ [".hidden", "haskell.png"] _ = return $ W.responseLBS H.status200 [("Content-Type", "image/png")] $ L.fromChunks [$(embedFile "images/haskell.png")]
 staticAppPieces ss rawPieces req = liftIO $ do
     case toPieces rawPieces of
         Just pieces -> checkPieces ss pieces req >>= response
-- 
1.8.2.rc3

